docs/examples/stocks-portfolio.md

```diff
@@ -0,0 +1,404 @@
+   "tradeable":false,
+   "dividendYield":"None",
+   "bidSize":1800,
+   "dayHigh":156.92,
+   "exchange":"NYQ",
+   "shortName":"Unity Software Inc.",
+   "longName":"Unity Software Inc.",
+   "exchangeTimezoneName":"America/New_York",
+   "exchangeTimezoneShortName":"EST",
+   "isEsgPopulated":false,
+   "gmtOffSetMilliseconds":"-18000000",
+   "quoteType":"EQUITY",
+   "symbol":"U",
+   "messageBoardId":"finmb_241908542",
+   "market":"us_market",
+   "annualHoldingsTurnover":"None",
+   "enterpriseToRevenue":54.824,
+   "beta3Year":"None",
+   "profitMargins":-0.35116002,
+   "enterpriseToEbitda":-213.991,
+   "52WeekChange":1.1919534,
+   "morningStarRiskRating":"None",
+   "forwardEps":"None",
+   "revenueQuarterlyGrowth":"None",
+   "sharesOutstanding":270776992,
+   "fundInceptionDate":"None",
+   "annualReportExpenseRatio":"None",
+   "bookValue":"None",
+   "sharesShort":6390454,
+   "sharesPercentSharesOut":0.023599999,
+   "fundFamily":"None",
+   "lastFiscalYearEnd":1577750400,
+   "heldPercentInstitutions":0.61965,
+   "netIncomeToCommon":-317163008,
+   "trailingEps":"None",
+   "lastDividendValue":"None",
+   "SandP52WeekChange":0.14322305,
+   "priceToBook":"None",
+   "heldPercentInsiders":0.16246,
+   "nextFiscalYearEnd":1640908800,
+   "mostRecentQuarter":1601424000,
+   "shortRatio":3.2,
+   "sharesShortPreviousMonthDate":1607990400,
+   "floatShares":120820703,
+   "enterpriseValue":38934634496,
+   "threeYearAverageReturn":"None",
+   "lastSplitDate":"None",
+   "lastSplitFactor":"None",
+   "legalType":"None",
+   "lastDividendDate":"None",
+   "morningStarOverallRating":"None",
+   "earningsQuarterlyGrowth":"None",
+   "dateShortInterest":1610668800,
+   "pegRatio":-15.31,
+   "lastCapGain":"None",
+   "shortPercentOfFloat":0.048600003,
+   "sharesShortPriorMonth":5086368,
+   "impliedSharesOutstanding":"None",
+   "category":"None",
+   "fiveYearAverageReturn":"None",
+   "regularMarketPrice":152.33,
+   "logo_url":"https://logo.clearbit.com/unity.com"
+}
+```
+
+We want the `['longName']`, `['regularMarketOpen']` and `['longBusinessSummary']`. Now we want the bot to send the data, so we will be using `await message.respond()`. Here is what it should look like:
+
+```python
+class StockSkill(Skill):
+
+    @match_regex(r"Stock: (.*)")
+    async def search_info_stock(self, message):
+        stock_name = message.regex.group(1)
+        stock = yf.Ticker(stock_name)
+        await message.respond(f"Name: {stock.info['longName']}")
+        await message.respond(f"Current Price: {stock.info['regularMarketOpen']} {stock.info['currency']}")
+        await message.respond(f"About: {stock.info['longBusinessSummary']}")
+```
+
+Congratulations! You have now made a bot that can tell you the name, price and info on a stock. You can do this with `Stock: <symbol-of-stock>`. With the basic part of the bot finished we can make the portfolio/saving section.
+
+### Saving and Adding to the Portfolio
+
+#### Creating the Table
```

Comment:
Like I said before, i'm happy to have the SQL steps here -  if you want to replace it you could change this step like:

### Putting in opsdroid memory

Opsdroid has a memory class which can be used to persist data, to use opsdroid memory you need to add the following to your configuration:

```yaml
databases:
  sqlite:
```

Then we can add to the database using `await self.opsdroid.memory.put(stocks, params)`

---

You might need to think about how the data is going to be saved/returned

Reply:
Thank you this info really helps!